symplyphysics/laws/thermodynamics/infinitesimal_work_in_quasistatic_process.py

```diff
@@ -1,41 +1,69 @@
+r"""
+Infinitesimal work in quasistatic process
+=========================================
+
+For a process in a closed system, occuring slowly enough for it to be quasi-static (i.e. the system
+remains in internal physical, but not necessarily chemical, thermodynamic equilibrium), the infinitesimal
+increment of work done *by* the system is related to the pressure inside the system and the infinitesimal
+increment of volume of the system.
+
+**Notation:**
+
+#. :math:`\delta` (:code:`delta`) denotes that the increment is an inexact, path-dependent differential.
+#. :math:`d` denotes that the increment is an exact, path-independent differential.
+"""
+
 from sympy import Eq
 from symplyphysics import (
     units,
     Quantity,
     Symbol,
-    print_expression,
     validate_input,
     validate_output,
 )
 
-# Description
-## For a process in a closed system, occuring slowly enough for it to be quasi-static (i.e. the system
-## remains in internal physical, but not necessarily chemical, thermodynamic equilibrium), the infinitesimal
-## increment of work done _by_ the system is related to the pressure inside the system and the infinitesimal
-## increment of volume of the system.
+infinitesimal_work_done = Symbol("infinitesimal_work_done", units.energy)
+r"""
+Infinitesimal increment of work done *by* the system.
 
-# Law: delta W = P * dV
-## delta W - an infinitesimal increment of work done _by_ the system
-## P - pressure inside the system
-## dV - infinitesimal increment of volume of the system
+Symbol:
+    :code:`delta W`
 
-# Note:
-## - `delta` means that the increment is an inexact (path-dependent) differential.
-## - `d` means that the increment is an exact (path-independent) differential.
+Latex:
+    :math:`\delta W`
+"""
+
+pressure = Symbol("pressure", units.pressure)
+"""
+Pressure inside the system.
+
+Symbol:
+    :code:`p`
+"""
 
-infinitesimal_work_done = Symbol("infinitesimal_work_done", units.energy)
-pressure_inside_system = Symbol("pressure_inside_system", units.pressure)
 infinitesimal_volume_change = Symbol("infinitesimal_volume_change", units.volume)
+"""
+Infinitesimal increment of volume of the system.
+
+Symbol:
+    :code:`dV`
 
-law = Eq(infinitesimal_work_done, pressure_inside_system * infinitesimal_volume_change)
+Latex:
+    :math:`dV`
+"""
 
+law = Eq(infinitesimal_work_done, pressure * infinitesimal_volume_change)
+r"""
+:code:`delta W = p * dV`
 
-def print_law() -> str:
-    return print_expression(law)
+Latex:
+    .. math::
+        \delta W = p \, dV
```

Comment:
Why comma here?

Reply:
It adds a little bit of space between `p` and `dV` so that they don't look smudged together. In my experience, it's quite common to do that.